ed25519/src/seed.cpp

```diff
@@ -15,7 +22,12 @@
 #endif
 
 void ed25519_create_seed(unsigned char *seed) {
-#ifdef _WIN32
+#ifdef TORRENT_WINRT
+    IBuffer^ seedBuffer = CryptographicBuffer::GenerateRandom(32);
```

Comment:
is it mandatory to use c++/cx with the winrt API? basically, is it a safe assumption that when we're on WinRT we're also using a different c++ variant?